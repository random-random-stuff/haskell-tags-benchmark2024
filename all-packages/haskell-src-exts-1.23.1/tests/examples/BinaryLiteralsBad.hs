-- Missing binary literals extension. Should fail.
f :: Int -> ()
f 0b0 = ()
f _   = ()
