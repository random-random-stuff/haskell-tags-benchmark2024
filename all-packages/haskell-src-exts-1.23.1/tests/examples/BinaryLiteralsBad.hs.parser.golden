ParseFailed
  (SrcLoc "tests/examples/BinaryLiteralsBad.hs" 3 1)
  "arity mismatch for 'f'"
